clients/unshaded/src/main/java/tachyon/client/WorkerNetAddress.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.client;
+
+import com.google.common.base.Objects;
+
+import tachyon.annotation.PublicApi;
+
+/**
+ * The network of a worker.
+ */
+@PublicApi
+public final class WorkerNetAddress {
+  private final String mHost;
+  private final int mRpcPort;
+  private final int mDataPort;
+  private final int mWebPort;
+
+  /**
+   * Constructs the worker net address.
+   */
+  public WorkerNetAddress(String host, int rpcPort, int dataPort, int webPort) {
```

Comment:
We could pick either approach. But this constructor is still useful as in tests.